sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2922,6 +2922,15 @@ object SQLConf {
     .stringConf
     .createWithDefault("")
 
+  val LEGACY_CREATE_HIVE_TABLE_BY_DEFAULT =
```

Comment:
This seems to be inconsistent because the other code expects `LEGACY_CREATE_HIVE_TABLE_BY_DEFAULT_ENABLED`.